content/blog/2020/06/2020-06-14-repellent-terminology.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Repellent Terminology"
+tags:
+- community
+- opensource
+author: markyjackson-taulia
+description: "Terminology Updates"
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We have to talk, a real conversation about the power of words, honest talk about hurtful words, and the energy they hold.
```

Comment:
consider:
We have to talk—a real conversation about the power of words and the hurt and emotion they can carry.